wikitongues/wikitongues/config/load_configs.py

```diff
@@ -0,0 +1,43 @@
+import sys
+from sys import platform
+import configparser
+import os
+
+
+def load_configs():
+
+    print("loading config file")
+
+    default_config = configparser.ConfigParser()
+    default_config.read_file(open('config/indexing.cfg'))
+    local_config_paths = default_config.items('local_config_path')
+
+    user_config = configparser.ConfigParser()
+
+    if platform == "win32":
+        env = os.getenv('APPDATA')
+        env += local_config_paths[0][1]
+    elif platform == "linux" or platform == "linux2":
```

Comment:
Can you check for Mac here too? Confirmed that `os.getenv('HOME')` works on my Mac

Reply:
Added a check for mac. Can you confirm it works?